.changeset/curly-turkeys-drum.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': patch
+---
+
+Adjusted flat and factory section headings to use medium weight.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)